.github/workflows/_ci.yaml

```diff
@@ -465,6 +489,30 @@ jobs:
     with:
       PAX_IMAGE: ${{ needs.build-rosetta-pax.outputs.DOCKER_TAG_FINAL }}
     secrets: inherit
+  
+  test-gemma:
+    needs: build-gemma
+    uses: ./.github/workflows/_test_unit.yaml  
+    if: inputs.ARCHITECTURE == 'amd64'
+    with:
+      TEST_NAME: gemma
+      EXECUTE: |
+        docker run --shm-size=1g --gpus all ${{ needs.build-gemma.outputs.DOCKER_TAG_FINAL }} \
+        bash -ec \
+        "pip install -e /opt/gemma[test] && pytest /opt/gemma/ && python /opt/gemma/tests/test_paligemma.py" | tee test-gemma.log
```

Comment:
The pip install seems odd here. I guess the point is to install extra test dependencies?

Reply:
Yes ([proof1](https://github.com/google-deepmind/gemma/blob/main/pyproject.toml#L31), [proof2](https://github.com/google-deepmind/gemma?tab=readme-ov-file#running-the-unit-tests)).